flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -207,9 +207,12 @@ public void testCleanUpExceptionSuppressing() throws Exception {
 			testHarness.waitForTaskCompletion();
 		}
 		catch (Exception ex) {
-			if (!ExceptionUtils.findThrowable(ex, ExpectedTestException.class).isPresent()) {
-				throw ex;
-			}
+			Assert.assertTrue(ex.getCause() instanceof ExpectedTestException);
+			Assert.assertTrue(
```

Comment:
If you replace the 'assertTrue' with 'assertEquals' as suggested by IDE, it will then ask to simplify the code.

I think 'assertTrue' is better than 'assertEquals' since we can achieve the same purpose with shorter code.

Reply:
I'm fine with either option, but just to clarify: do you mean the number of characters in `assertEquals`?

Reply:
I mean

`Assert.assertTrue(ex.getCause() instanceof ExpectedTestException)`; vs.
`Assert. assertEquals(True, ex.getCause() instanceof ExpectedTestException)`;

That is the IDE suggesting.